src/blocks/inline-notice/style.scss

```
@@ -32,7 +32,7 @@
 
 	.uagb-notice-title {
 		margin: 0;
-		width: -webkit-fill-available;
+		width: -moz-available;
```

Let's add the browser based overrides.

```suggestion
		width: -moz-available;
		width: -webkit-fill-available;
		width: fill-available;
```

---

Added browser based overrides.